components/notif-card/index.tsx

```diff
@@ -73,6 +73,26 @@ const NotifCard: FunctionComponent<NotifCardProps> = ({ id, type }) => {
                   </div>
                 );
               })}
+            {type === 'desktop' && (
+              <>
+                <p>There are prerequisites for using Desktop notifications:</p>
```

Comment:
i think it's singular: notification without `s`

Reply:
Thanks man!